docs/src/app/components/CodeExample/CodeBlockTitle.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import IconButton from 'material-ui/lib/icon-button';
+import CodeIcon from 'material-ui/lib/svg-icons/action/code';
+import Toolbar from 'material-ui/lib/toolbar/toolbar';
+import ToolbarGroup from 'material-ui/lib/toolbar/toolbar-group';
+import ToolbarTitle from 'material-ui/lib/toolbar/toolbar-title';
+
+const CodeBlockTitle = (props) => (
+  <Toolbar>
+    <ToolbarGroup float="left">
+      <ToolbarTitle text={props.title || 'Example'} />
+    </ToolbarGroup>
+    <ToolbarGroup float="right">
+      <IconButton touch={true}>
```

Comment:
Can we add a tooltip, like `View source`.
I don't want people opening issues to ask how to show the source code :grin:.